app/scripts/filters/resources.js

```diff
@@ -138,23 +138,23 @@ angular.module('openshiftConsole')
   })
   .filter('canIDoAny', function(canIFilter) {
     var resourceRulesMap = {
-      "buildConfigs": {"buildconfigs": ["delete", "update"], "buildconfigs/instantiate": ["create"]},
+      "buildConfigs": {"buildconfigs": {verbs: ["delete", "update"]}, "buildconfigs/instantiate": {verbs: ["create"]}},
       "builds": {"builds/clone": ["create"], "builds": ["delete", "update"]},
-      "deploymentConfigs": {"extensions/horizontalpodautoscalers": ["create", "update"], "deploymentconfigs": ["create", "update"]},
-      "deployments": {"replicationcontrollers": ["update", "delete"]},
-      "horizontalPodAutoscalers": {"extensions/horizontalpodautoscalers": ["update", "delete"]},
-      "imageStreams": {"imagestreams": ["update", "delete"]},
-      "persistentVolumeClaims": {"persistentvolumeclaims": ["update", "delete"]},
-      "pods": {"pods": ["update", "delete"], "deploymentconfigs": ["update"]},
-      "replicationControllers": {"horizontalpodautoscalers": ["create", "update"], "replicationcontrollers": ["create", "update"]},
-      "routes": {"routes": ["update", "delete"]},
-      "services": {"services": ["update", "create", "delete"]},
-      "projects": {'projects': ['delete', 'update']}
+      "deploymentConfigs": {"horizontalpodautoscalers": {verbs: ["create", "update"], group: "autoscaling"}, "deploymentconfigs": {verbs: ["create", "update"]}},
```

Comment:
no reason for the value here to be a map... putting group and verbs together is weird... this would make more sense:
```
"deploymentConfigs: [
  {group:'autoscaling', resource:'horizontalpodautoscalers', verbs: ["create","update"]},
  {group:'',            resource:'deploymentconfigs',        verbs: ["create", "update"]},
],
...
      return _.some(resourceRulesMap[resource], function(rules) {
        return _.some(rules.verbs, function(verb) {
          return canIFilter(rules.resource,verb, {group: (rules.group || "")});
         });
       });
```